Explore the mapping scores (#835)
AlexsLemonade/OpenScPCA-analysis: analyses/cell-type-wilms-tumor-06/notebook/04_annotation_Across_Samples_exploration.Rmd

Reviewer: A general comment here: Can you add _text_ above plots stating that the line is drawn at the threshold being explored in the notebook?

              options = list(  dom = 'Bfrtip',
                               buttons = c( 'csv', 'excel')))
```
### Label transfer (mapping) scores for the four compartments

```{r fig.height=5, fig.width=10, message=FALSE, warning=FALSE, out.width='100%'}
p <- do_Feature_densityplot( df = cell_type_df,
                    feature = "compartment_score",
                    group.by = "compartment"
                    )
p +  geom_vline(xintercept=params$mapping_score_thr)
```

> Some cells have an excellent mapping score (>0,85) while other performed worth. Let's have a look at the repartiction per patient. 

Reviewer: I think this is what you mean?
```suggestion
Some cells have an excellent mapping score (>0,85) while other performed worse. Let's have a look at the compartments per patient. 
```

```{r fig.height=15, fig.width=15, message=FALSE, warning=FALSE, out.width='100%'}
p <- do_Feature_boxplot( df = cell_type_df,
                    feature = "compartment_score",
                  group.by = "compartment",
                    split.by = "sample_id")

> p + geom_hline(yintercept = params$mapping_score_thr)

Reviewer: I suggest that we set the y-axes to be identical for these boxplots since we want to easily compare values on the same scale across panels. This way, the heights will be identical, and a lower limit of 0.25 will be enough to capture all points.

```suggestion
p + geom_hline(yintercept = params$mapping_score_thr)  + coord_cartesian(ylim = c(0.25,1))
```

```

Regarding the endothelial cells, some patient have very high mapping scores while others perform poorly. 
Let's have a look if this can be linked to some metadata/clinical data.

##### Treatment

```{r fig.height=3, fig.width=10, message=FALSE, warning=FALSE, out.width='100%'}
p <- do_Feature_boxplot( df = cell_type_df,
                    feature = "compartment_score",
                  group.by = "compartment",
                    split.by = "treatment")

> p + geom_hline(yintercept = params$mapping_score_thr)

Reviewer: ```suggestion
p + geom_hline(yintercept = params$mapping_score_thr) + coord_cartesian(ylim = c(0.25,1))
```

```

##### Subdiagnosis

```{r fig.height=3, fig.width=10, message=FALSE, warning=FALSE, out.width='100%'}
p <- do_Feature_boxplot( df = cell_type_df,
                    feature = "compartment_score",
                  group.by = "compartment",
                    split.by = "subdiagnosis")

> p + geom_hline(yintercept = params$mapping_score_thr)

Reviewer: ```suggestion
p + geom_hline(yintercept = params$mapping_score_thr) + coord_cartesian(ylim = c(0.25,1))
```

```

##### Disease timing

```{r fig.height=3, fig.width=15, message=FALSE, warning=FALSE, out.width='100%'}
p <- do_Feature_boxplot( df = cell_type_df,
                    feature = "compartment_score",
                  group.by = "compartment",
                    split.by = "disease_timing")

> p + geom_hline(yintercept = params$mapping_score_thr)

Reviewer: ```suggestion
p + geom_hline(yintercept = params$mapping_score_thr) + coord_cartesian(ylim = c(0.25,1))
```
